test/system/test_misc.py

```diff
@@ -121,6 +121,38 @@ def test_wrong_value_for_bc_field(field):
         sim.initialise()
 
 
+def test_txt_export_desired_times(tmp_path):
+    """
+    Tests that TXTExport can be exported at all timesteps
+    """
+    my_model = F.Simulation()
+
+    my_model.mesh = F.MeshFromVertices(np.linspace(0, 1))
+    my_model.materials = F.Material(1, 1, 0)
+    my_model.settings = F.Settings(1e-10, 1e-10, final_time=1)
+    my_model.T = F.Temperature(500)
+    my_model.dt = F.Stepsize(0.1)
+
+    my_export = F.TXTExport(
+        "solute", label="mobile_conc", times=[0.2, 0.5], folder=tmp_path
+    )
+    my_model.exports = [my_export]
+
+    my_model.initialise()
+    my_model.run()
```

Comment:
Is a way to test the export without having to make a system test? but I'm not sure on this

Reply:
There must be a way to just call `write()` but I don't know if it's worth it, this does the trick